ports/cairo/portfile.cmake

```
@@ -25,8 +24,13 @@ if ("x11" IN_LIST FEATURES)
     message(WARNING "You will need to install Xorg dependencies to use feature x11:\napt install libx11-dev libxft-dev\n")
 endif()
 
+if("gobject" IN_LIST FEATURES)
+    message(WARNING "Feature gobject currently only supports dynamic build.")
```

```suggestion
    message(FATAL_ERROR "Feature gobject currently only supports dynamic build.")
```
I think we should stop the build process here.